src/modules/cmdpal/Microsoft.CmdPal.UI/MainWindow.xaml.cs

```diff
@@ -293,13 +293,25 @@ public void Receive(QuitMessage message)
     public void Receive(DismissMessage message) =>
         PInvoke.ShowWindow(_hwnd, SHOW_WINDOW_CMD.SW_HIDE);
 
+    public void Receive(SystemTrayIconChangedMessage message)
+    {
+        if (message.Enabled)
```

Comment:
Rather than adding another message, would it be easier to just handle this in `HotReloadSettings`? 

I suppose this is more surgical than just "the whole settings changed"